bin/settings.toml

```diff
@@ -0,0 +1,104 @@
+##
+# Open Space Program
+# Copyright © 2019-2020 Open Space Program Project
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+##
+
+[vehicle_pitch_up]
+primary = "S"
+secondary = "None"
+invert = true
+
+[vehicle_pitch_dn]
+primary = "W"
+secondary = "None"
+invert = true
+
+[vehicle_yaw_lf]
+primary = "A"
+secondary = "None"
+invert = true
+
+[vehicle_yaw_rt]
+primary = "D"
+secondary = "None"
+invert = true
```

Comment:
what is "invert" ?

Reply:
It's supposed to be holdable, I made a mistake while reading the code